docker-compose.yml

```diff
@@ -1,12 +1,24 @@
 web:
+  restart: always
   build: .
+  command: grunt
   links:
    - db
   ports:
    - "3000:3000"
-  environment: 
-   NODE_ENV: development
+   - "5858:5858"
+   - "35729:35729"
+  environment:
+   - NODE_ENV=development
+  volumes:
+   - ./:/opt/mean.js
+   - /opt/mean.js/node_modules
+   - /opt/mean.js/public
+   - /opt/mean.js/uploads
 db:
-  image: mongo
-  ports: 
+  restart: always
+  image: mongo:latest
```

Comment:
Actually seems like different 3.x versions might also break stuff (#1246) so using tag `3.2` might be ideal.